lib/Service.php

```diff
@@ -189,6 +189,27 @@ function write($rootElementName, $value, $contextUri = null) {
 
     }
 
+    /**
+     * Maps the given plain-old-php-object (popo) to the given rootElementName within the given namespace.
```

Comment:
I think I would lean towards ValueObject, it also sounds a bit better and I feel it requires a bit less explanation...